[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/planning-the-environment-for-biztalk-server.md

 [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] also includes several other features that facilitate business integration including a message workflow engine, a Business Rule Engine (BRE), and technologies for information workers such as Business Activity Monitoring (BAM).

>  For more information about using [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] business process management functionality, see White Paper: Microsoft and BPM—Technical Overview. To know more about the different integration technologies offered by Microsoft and the advantages one has over the other, see [Understanding Microsoft Integration Technologies](https://go.microsoft.com/fwlink/?LinkId=158452).

Reviewer: ```suggestion
 For more information about using [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] business process management functionality, see [Microsoft and BPM - A Technology Overview](https://download.microsoft.com/documents/australia/biztalk/post_event/Whitepaper_MicrosoftandBPM.pdf). To learn more about the different integration technologies offered by Microsoft and the advantages one has over the other, see [Why migrate from BizTalk Server to Azure Integration Services](/azure/logic-apps/biztalk-server-to-azure-integration-services-overview).
```

 Certain integration scenarios are better suited to other Microsoft products. If your primary focus is upon any of the following scenarios, consider using these Microsoft products instead of [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)]:

|**Scenario**|**Product to use**|
|------------------|------------------------|

> |User provisioning|**Microsoft Identity Lifecycle Manager 2010**<br /><br /> For more information about Microsoft Identity Lifecycle Manager 2010, see [Microsoft Identity Lifecycle Manager 2010 FP1](https://go.microsoft.com/fwlink/?LinkId=204577).|

Reviewer: ```suggestion
|User provisioning|**Microsoft Identity Lifecycle Manager 2010**<br /><br /> For more information about Microsoft Identity Lifecycle Manager 2010, see [Microsoft Identity Lifecycle Manager 2010 FP1](/previous-versions/windows/desktop/forefront-2010/ee652374(v=vs.100)).|
```

- **RFID Enterprise** - Designed to provide a scalable, extensible platform for development, deployment, and management of rich RFID and sensor solutions, includes BizTalk RFID Server and BizTalk RFID Mobile.

>   For more information about the different editions of BizTalk Server, see Microsoft BizTalk Server Editions.

Reviewer: ```suggestion
  For more information about the different editions of BizTalk Server, see [Microsoft BizTalk Server Editions](https://download.microsoft.com/download/0/1/9/0194b3c5-72cc-4dcb-91a8-1908642c8fde/BizTalk_Server_2020_Licensing_Datasheet_EN_US.pdf).
```
